b2/data_source_b2_bucket.go

```diff
@@ -69,7 +69,7 @@ func dataSourceB2Bucket() *schema.Resource {
 			"default_server_side_encryption": {
 				Description: "The default server-side encryption settings of this bucket.",
 				Type:        schema.TypeList,
-				Elem:        getDataSourceDefaultServerSideEncryption(),
+				Elem:        getResourceDefaultBucketServerSideEncryption(),
```

Comment:
Is this works for date sources as you don't have `Computed: true` in `getResourceDefaultBucketServerSideEncryption`?